### PR TITLE
importable data for emulators

### PR DESCRIPTION
**Pull request by Maahi10001**
user and orgs data(dummy) of all emulators to use as an import file to perform import and export operations

## Description

The file named "testdata" includes some examples users, orgs and codelabz  data of respective local emulators the new users who are using the website can use these files data to import into emulators and experience it.

## Related Issue

#234 

## How Has This Been Tested?

importing from testdata

image

image

image

## Screenshots or GIF (In case of UI changes):

video on testing the file :-  https://recordit.co/ztBhmUbKpl

imported  user data from the file in emulator

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: testdata/storage_export/buckets.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "buckets": [
+    {
+      "id": "codelabz-966e5.appspot.com"
```
Comment (Niweera) [inline testdata/storage_export/buckets.json:4]: is this your personal bucket configuration?

Comment (Maahi10001) [reply testdata/storage_export/buckets.json:4]: > is this your personal bucket configuration?

yes

Comment (Niweera) [reply testdata/storage_export/buckets.json:4]: so is it okay to others to use it?

Comment (Maahi10001) [reply testdata/storage_export/buckets.json:4]: > so is it okay to others to use it?

yes sir,firebase bucket files are used to share data only I guess

Comment (Niweera) [reply testdata/storage_export/buckets.json:4]: ah sure

Comment (Maahi10001) [reply testdata/storage_export/buckets.json:4]: > so is it okay to others to use it?

and more over it is storage emulator's meta data like profile photo so may be a problem.

**Review by Niweera — APPROVED** [review]

**Pull request merged**